frontend/src/pages/DraftForm.tsx

```
@@ -0,0 +1,218 @@
+import { LoadingButton } from '@mui/lab'
+import { Box, Button, Typography } from '@mui/material'
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { useNavigate, useParams } from 'react-router'
+import useSWR from 'swr'
+import { EditIcon, SplitscreenIcon, VisibilityIcon } from '../components/icons'
+import { Error } from '../components/utilities/Error'
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)